inductiva/molecules/examples.py

```diff
@@ -0,0 +1,36 @@
+"Set of functions to load examples of molecules."
```

Comment:
This is super cool! Please just notice if it works on Colab due to drive mounts - we may still need to do them, but it is important just to be aware.

Reply:
![image](https://github.com/inductiva/inductiva/assets/114397668/c780c0f9-c0c1-4d0c-b394-977dce9fc042)
It does :)